### PR TITLE
Sponsorship page, prospectus, and landing page section

### PR DESCRIPTION
**Pull request by rpcme**
This PR has the page, landing page update, and pdf for posting.

### CODE REVIEW

File: templates/2015/sponsoring.html.ep [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <p>
+        Mojoconf is an annual conference that brings together WebDev and WebOps
+        professionals from around the globe. It focuses on the Mojolicious web
+        development framework, which enables developers and designers to move
+        quickly from proof-of-concept to production quality web applications.
+      </p>
+
+      <h3><i class="fa fa-map-marker"></i>Get the Prospectus</h3>
+      <p>
+        The <%= link_to '/2015/prospectus.pdf' => ( target => '_blank' ) => begin %>
```
Comment (jhthorsen) [inline templates/2015/sponsoring.html.ep:22]: Do you need target blank? Is that some magical feature where the pdf is loaded inside the browser maybe?

Comment (rpcme) [inline templates/2015/sponsoring.html.ep:22]: I just do it to open in new tab.

File: templates/2015/sponsoring.html.ep [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+%= include 'partial/top_image';
+
+<div class="content-wrapper">
+  %= include 'partial/mojoconf-content-header', header => 'Sponsoring Mojoconf'
+  <div class="content">
+    <section>
+
+      <h3><i class="fa fa-map-marker"></i>About Mojoconf</h3>
+      <p>
+        Mojoconf is an annual conference that brings together WebDev and WebOps
```
Comment (jhthorsen) [inline templates/2015/sponsoring.html.ep:14]: WebDev or WebDevs?

File: templates/2015/sponsoring.html.ep [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      <p>
+        With your sponsorship, we give you direct exposure to attendees in the
+        WebDev and WebOps space. Whether it’s recruiting, development tools,
+        cloud services, or another relevant offering, sponsorship will highlight
+        your deep interest and commitment to attendee interests.
+      </p>
+
+      <h3><i class="fa fa-map-marker"></i>Focused Attention</h3>
+      <p>
+        The conference size means more time for you to build relationships with
```
Comment (jhthorsen) [inline templates/2015/sponsoring.html.ep:37]: Is this really a selling point?

**Comment by jhthorsen** [comment]
LGTM. Just merge it with master and the tests should be OK as well.

**Pull request merged**